src/lib/utils.ts

```
@@ -1,6 +1,6 @@
-import { type ClassValue, clsx } from "clsx";
-import { twMerge } from "tailwind-merge";
+import { clsx, type ClassValue } from "clsx"
+import { twMerge } from "tailwind-merge"
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)